apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJob.java

```diff
@@ -67,8 +69,11 @@
 public final class CcwRifLoadJob implements PipelineJob {
   private static final Logger LOGGER = LoggerFactory.getLogger(CcwRifLoadJob.class);
 
-  /** Shortcut for calculating GIGA (filesize). */
-  private static final int GIGA = 1000 * 1000 * 1000;
+  /**
+   * Minimum amount of free disk space (in bytes) to allow pre-fetch of second data set while
+   * current one is being processed.
+   */
+  public static final long MIN_BYTES_FOR_SECOND_DATA_SET_DOWNLOAD = 50 * FileUtils.ONE_GB;
```

Comment:
Burying the 50 down in the code and doing the extra multiplications to compute 1gb was less clear than just creating an appropriate constant for the total size.  Apache commons already had a constant for `gb that we can use.